test/utils/SafeTransferLib.t.sol

```diff
@@ -224,7 +226,8 @@ contract SafeTransferLibTest is Test {
         uint256 amount,
         bytes calldata brutalizeWith
     ) public brutalizeMemory(brutalizeWith) {
-        if (uint256(uint160(nonContract)) <= 18 || nonContract.code.length > 0) return;
+        vm.assume(uint256(uint160(nonContract)) > 18 && nonContract.code.length == 0);
+        vm.assume(nonContract != HEVM_ADDRESS && nonContract != CONSOLE);
```

Comment:
Could you use those constants from `forge-std` instead ?
The contract test inherit from `Test` imported from `forge-std`.
In `forge-std`, it should be accessible as `VM_ADDRESS` and `CONSOLE` directly.

Same comment for the 3 other tests concerned

Reply:
Hi @obatirou 
`CONSOLE` is undeclared in `Test`.
If you comment out the `CONSOLE` declaration from my code, you get this:
```
error[7576]: DeclarationError: Undeclared identifier.
   --> test/utils/SafeTransferLib.t.sol:349:61:
    |
349 |         vm.assume(recipient != HEVM_ADDRESS && recipient != CONSOLE);
    |                                                             ^^^^^^^
```
And if you replace `CONSOLE` with `CONSOLE_ADDRESS`, you get this:
```
error[7576]: DeclarationError: Undeclared identifier.
   --> test/utils/SafeTransferLib.t.sol:349:61:
    |
349 |         vm.assume(recipient != HEVM_ADDRESS && recipient != CONSOLE_ADDRESS);
    |   
```
I believe this is because although `Script` (and therefore `Test`) imports `console.sol` and `console2.sol`, the constant `CONSOLE_ADDRESS` address only gets declared within the contract and not at the file level, so it never finds its way into `Test`.

Reply:
Those constants are declared in `CommonBase`
If you follow the inheritance in [forge-std/src/Test.sol](https://github.com/foundry-rs/forge-std/blob/master/src/Test.sol): 
`Test` -> `TestBase` -> `CommonBase` so they should be accessible when your test contract is inheriting from `Test`
It seems that your branch is not up to date with `dev`, I think that is why is it not working
You might want to rebase

Reply:
Nice, you should be able to now change `HEVM_ADDRESS` to `VM_ADDRESS`